### DISCUSSION TITLE
FAQ seems to be missing a key piece of information

### DISCUSSION THREAD

**Discussion by fabnavigator** [pivpn/pivpn#1776] [Q&A] [answered]
I'm having installation issues and have followed the ```https://docs.pivpn.io/faq/``` instructions carefully.  I'm at the packet capture part, and the only unsuccessful situation that is noted in the FAQ is the total lack of packets. What if you see packets received by the server, but no packets are returned to the client? That seems like a situation that should be mentioned. It is my situation. I assume there is some additional debugging I can do to find out why the server didn't return a packet to my client.

**Comment by orazioedoardo** [comment]
>  What if you see packets received by the server, but no packets are returned to the client?

Then either a bot is scanning the server or the client or the server has misconfigured keys, but the latter shouldn't happen with the script.

**Reply by fabnavigator** [reply]
You are correct. There were misconfigured keys. I can't blame the script, but a simple statement like what you wrote above in the FAQ would have helped me.

**Reply by orazioedoardo** [reply, marked as answer]
I've updated the docs accordingly: https://github.com/pivpn/docs/commit/05c704ba71fe0fde8943ab6f4302e7e81880098f

**Reply by fabnavigator** [reply]
Thank you.
